digid_eherkenning/oidc/admin.py

```diff
@@ -90,34 +90,47 @@ def fieldsets_factory(claim_mapping_fields: Sequence[str]):
 
 
 @admin.register(DigiDConfig)
-class OpenIDConnectConfigDigiDAdmin(SingletonModelAdmin):
+class DigiDConfigAdmin(SingletonModelAdmin):
     form = admin_modelform_factory(DigiDConfig)
-    fieldsets = fieldsets_factory(claim_mapping_fields=["identifier_claim_name"])
+    fieldsets = fieldsets_factory(claim_mapping_fields=["bsn_claim"])
 
 
 @admin.register(EHerkenningConfig)
-class OpenIDConnectConfigEHerkenningAdmin(SingletonModelAdmin):
+class EHerkenningConfigAdmin(SingletonModelAdmin):
     form = admin_modelform_factory(EHerkenningConfig)
-    fieldsets = fieldsets_factory(claim_mapping_fields=["identifier_claim_name"])
+    fieldsets = fieldsets_factory(
+        claim_mapping_fields=[
+            "identifier_type_claim",
+            "legal_subject_claim",
+            "branch_number_claim",
+            "acting_subject_claim",
```

Comment:
I might be misunderstanding this, but isn't the distinction between `legal_subject_claim`/`acting_subject_claim` only for machtigen?

Reply:
so, for an eherkenning login *not* for machtigen, there's always a personal "middel", which ties a physical person (employee) to the company and allows them to act on behalf of the company they are representing. In this case, the acting subject is the person/employee, and the legal subject is the company they work for

adding the machtigen layer adds another claim: the one holding the representee identifier (BSN or KVK, typically, depending on if they are doing stuff for a person or other company)

Reply:
Ah I see, thanks for the explanation!